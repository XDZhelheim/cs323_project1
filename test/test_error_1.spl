int test_error_1(int a int b) {
    char c = ?;
}