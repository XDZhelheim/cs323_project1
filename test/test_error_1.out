Error type B at Line 1: Missing comma ','
Error type A at Line 2: unknown lexeme ?
